nominees/data-observation-toolkit.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Data Observation Toolkit",
+  "aliases": [
+    "DOT"
```

Comment:
I am not sure if this is a relevant alias - seems a bit too generic. I don't know that most people would associate 'DOT' with this project.

Reply:
Fixed in commit 9dc09b2